app/src/main/res/values-en/strings.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="sign_in">Sign In</string>
+    <string name="sign_in_button">SIGN IN</string>
+    <string name="user_name">User name</string>
+    <string name="forgot_password">Forgot password?</string>
+    <string name="settings">Settings</string>
+    <string name="cancel">cancel</string>
+    <string name="save">save</string>
+    <string name="base_url">Base URL</string>
+    <string name="password">Password</string>
+    <string name="base_url_must_not_be_empty">Base URL must not be empty.</string>
+    <string name="saved_successfully">Saved successfully.</string>
```

Comment:
Please check the latest code of main and include them into this pull request. eg. there is no . at then end of messages.